src/Http/Organizer/LegacyContactPointRequestBodyParser.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Organizer;
+
+use CultuurNet\UDB3\Http\Request\Body\RequestBodyParser;
+use Psr\Http\Message\ServerRequestInterface;
+
+class LegacyContactPointRequestBodyParser implements RequestBodyParser
```

Comment:
Nit: In other places this is usually called `BackwardCompatible...RequestBodyParser`. Either name is fine by me but maybe rename the others if you prefer `Legacy...RequestBodyParser`.

Reply:
Nicely spotted. This was indeed a sub comment inside this commit https://github.com/cultuurnet/udb3-silex/pull/791/commits/5bd28e03cc52a4beb5a470d5cc14325b31d708e9

Reply:
Ah I overlooked that because I reviewed the complete diff now for brevity's sake 😉

Reply:
But both names are fine for me. You can pick what you prefer 😄

Reply:
As discussed on Slack I took the `Legacy` variant